modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -326,6 +326,26 @@ public void testAddCustomObjectField() throws Exception {
 						Collections.emptyList()
 					).build())));
 
+		_assertFailure(
+			ObjectFieldSettingValueException.MissingRequiredValues.class,
+			"The settings timeStorage are required for object field datetime",
```

Comment:
Could you reorder this block of code, please?
I think it will be after the failure "The settings maxLength are required for object field text".